src/test/end-to-end/pages/uploadDocument/uploadDocument.js

```diff
@@ -4,14 +4,32 @@ const commonConfig = require('../../data/commonConfig.json');
 module.exports = async function () {
 
     const I = this;
-    await I.see('Upload Document');
-    await I.click(commonConfig.addNewButton);
-    await I.attachFile('#documentCollection_0_uploadedDocument', 'data/fileUpload.txt');
-    await I.wait(5);
-    await I.fillField('#documentCollection_0_shortDescription', commonConfig.shortDescription);
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.submit);
-    await I.wait(5);
-    await I.click("//div[text()='Documents']");
-    await I.see('fileUpload.txt');
+    //Start of Upload document page
+    I.see('Upload Document');
+    I.see('Case Number:');
+    I.see('Case documentation (Optional)');
+    I.see('Add new');
+    I.see('Upload documentation for the case');
+
+    I.click("//button[@class='button write-collection-add-item__top']");
```

Comment:
Sometimes it's more readable to use Buttons Names with Clear Text than give XPath Locators.

Reply:
Used button name with clear text